tests/python-pytest/onnx/test_onnx.py

```diff
@@ -0,0 +1,86 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Tests for individual operators"""
```

Comment:
This is not using the onnx tests, right? In that case, please move them to the other folder and use nosetests since that framework is the default for now

Reply:
I will move it.
This file actually contains tests which currently don't exist on the ONNX repo. Once we make a couple of PRs on ONNX repo and they get merged, this file will get EOLed.

Reply:
Ah I see. If the plan is to integrate them into ONNX, then it's fine. Please just consider nosetests for tests that are specifically for the MXNet side